flink-metrics/flink-metrics-influxdb/src/main/java/org/apache/flink/metrics/influxdb/InfluxdbReporter.java

```diff
@@ -148,6 +148,6 @@ private static boolean isValidHost(String host) {
 	}
 
 	private static boolean isValidPort(int port) {
-		return 0 < port && port <= 65535;
+		return 0 <= port && port <= 65535;
```

Comment:
I think `0` is not a valid port in this case. This is being used to connect somewhere, where `0` makes sense only when setting up a server/listening socket.